scripts/helm/install.sh

```diff
@@ -72,6 +74,51 @@ die() {
   exit "${_return}"
 }
 
+prefix_args() {
+  local prefix="$1"
+  if [ -n "$prefix" ]; then
+    prefix="$prefix."
+  fi
+  local sep=""
+  for set in $(echo "${@:2}" | awk -F, '{ for(i=1; i<=NF; i++) print $i; }'); do
+    echo -n "$sep$prefix$set"
+    if [ -z "$sep" ]; then
+      sep=","
+    fi
+  done
+}
+
+set_args() {
+  echo -n "--set="
+  prefix_args "" "$@"
+}
+
+loki_args() {
+  local replicas=1
+  local minio="true"
+  local mode="standalone"
+  if [ "$replicas" != 1 ]; then
+    mode="distributed"
+  fi
+
+  if [ "$replicas" = "0" ] || [ "$INS_LOKI" = "false" ]; then
+    echo -n "$(set_args "loki.enabled=false,alloy.enabled=false")"
+    return 0
+  fi
+
+  echo -n "$(set_args "loki.loki.commonConfig.replication_factor=$replicas")" \
+          "$(set_args "loki.singleBinary.replicas=$replicas")" \
+          "$(set_args "loki.minio.mode=$mode") "
+
+  if [ "$minio" = "true" ]; then
+    echo -n "$(set_args "loki.minio.enabled=true,loki.minio.replicas=$replicas")"
+  else
+    echo -n "$(set_args "loki.loki.storage.type=filesystem")" \
+            "$(set_args "loki.singleBinary.persistence.enabled=true")" \
+            "$(set_args "loki.minio.enabled=false")"
```

Comment:
This might not be enough tbh, we need to set object_store as filesystem as well. You can check the loki-storage doc